api/VRDisplay.json

```diff
@@ -27,17 +27,30 @@
             "version_removed": "79"
           },
           "firefox": [
+            {
+              "version_added": "98",
+              "flags": [
+                {
+                  "type": "preference",
+                  "name": "dom.vr.enabled",
+                  "value_to_set": "true"
+                }
+              ]
+            },
             {
               "version_added": "55",
+              "version_removed": "98",
```

Comment:
@ddbeck Note, there are other ways that the older support might have been indicated. I have chosen just to mark these as removed because there isn't much point playing with an API that is "on the way out".

Reply:
As discussed in 1:1, there are many ways to indicate the legacy partial support in BCD. Rather than using any of these I plan to just follow the pattern of the existing BCD, because it isn't worth messing with that for an API that is on the way out.
Resolving